mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,410 @@
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+import warnings
+
+from ...utils import logger, verbose
+from ..utils import _read_segments_file
+from ..meas_info import _empty_info, create_info
+from ..base import _BaseRaw, _check_update_montage
+from ...channels.montage import Montage
+from ...epochs import _BaseEpochs
+from ...event import read_events
+from ...externals.six import string_types
+
+
+def _check_fname(fname):
+    """Check if the file extension is valid.
+    """
+    fmt = str(op.splitext(fname)[-1])
+    if fmt == '.dat':
+        msg = ('Old data format .dat detected. Please update your EEGLAB '
+               'version and resave the data in .fdt format')
+        raise NotImplementedError(msg)
+    elif fmt != '.fdt':
+        msg = ('Expected .fdt file format. Found %s format' % fmt)
+
+
+def _check_mat_struct(fname):
+    """Check if the mat struct contains 'EEG'.
+    """
+    from scipy import io
+    mat = io.whosmat(fname, struct_as_record=False,
```

Comment:
looks like you [missed my comment] (https://github.com/mne-tools/mne-python/pull/2676#discussion_r48028070) about checking the version when using the 0.12+ function